### PR TITLE
Combobox component features

### PR DESCRIPTION
**Pull request by Sengottaiyan-BMES**
Combobox component features

### CODE REVIEW

File: src/lib/components/Combobox/Combobox.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          <>
+            <div className="d-flex align-items-center m-1">
+              <Search className="ms-2 text-secondary search-icon" />
+              <input
+                type="text"
+                value={searchString}
+                onChange={(e) => setSearchString(e.target.value)}
+                placeholder="Search here"
+                className="w-100 px-2 py-2 bg-transparent focus:outline-none opacity-100"
+              />
```
Comment (anaeshBMES) [inline src/lib/components/Combobox/Combobox.tsx:57]: why are you not using BMES components?

File: src/lib/components/Combobox/Combobox.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import { Check, ChevronsUpDown, Search } from "lucide-react";
+import { Label, LabelVariants, Badge } from "../../../lib";
+import { cn } from "../../shared/components/DTOs";
+import { useEffect, useState } from "react";
+import { DropdownProps } from "./DTOs";
+import React from "react";
+import "./Combobox.scss";
+
+const Combobox = (props: DropdownProps) => {
+  const { label, items, placeholder = "Select", selectedValue, onValueChange, showCreateButton = true, createButton } = props;
```
Comment (anaeshBMES) [inline src/lib/components/Combobox/Combobox.tsx]: Change name into `onCreate`

File: src/lib/components/Combobox/Combobox.tsx (hunk 1 above)
Comment (Sengottaiyan-BMES) [reply src/lib/components/Combobox/Combobox.tsx:57]: width is not set properly

File: src/lib/components/Combobox/Combobox.tsx (hunk 2 above)
Comment (Sengottaiyan-BMES) [reply src/lib/components/Combobox/Combobox.tsx]: changed

**Pull request merged**